infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -100,8 +102,12 @@ public synchronized void handleMessageSetEpoch(@NonNull CorfuPayloadMsg<Long> ms
                                                    @NonNull IServerRouter r) {
         try {
             long epoch = msg.getPayload();
-            log.info("handleMessageSetEpoch: Received SET_EPOCH from {}, moving to new epoch {}",
-                    msg.getClientID(), epoch);
+            String remoteHostAddress = "0.0.0.0";
```

Comment:
why setting to 0.0.0.0 rather than null? remoteAddress() might return null anyways. If you want to differentiate between ctx.channel being null and ctx.channel being not connected then this is fine but not very expressive.

Reply:
I would prefer seeing zeros instead of emptiness)

Reply:
> If you want to differentiate between ctx.channel being null and ctx.channel being not connected then this is fine but not very expressive
Yes, it was just some way of distinguishing the channel being null.

Reply:
I have removed 0.0.0.0 thanks